### DISCUSSION TITLE
viewport

### DISCUSSION THREAD

**Discussion by smurf88** [jan-vandenberg/cruddiy#61] [Ideas]
love cruddiy, small contribution/ suggestion - add viewport to the head
`<meta name="viewport" content="width=device-width, initial-scale=1">`

**Comment by jan-vandenberg** [comment]
Where would this need to be added exactly and what would the benefit be?

**Reply by smurf88** [reply]
I would add it to template part- for everything that is not Ajax (might just check if Ajax inherits the parent viewport) Viewports sets the size for different devices and if you change the amount of columns for a small of larger screen it won’t looked zoomed out on smaller screens

**Reply by smurf88** [reply]
![prevwithout](https://user-images.githubusercontent.com/20774587/214764513-62f5e8d9-200a-4415-9f98-dbc2ddd9835d.png)
![prevwith](https://user-images.githubusercontent.com/20774587/214764648-722ea870-068a-4f67-a788-47569652a1c6.png)

1st screenshot is without and second is with.

**Comment by jan-vandenberg** [comment]
Aha like that. Cool, if you send a PR I'll try and merge it.

**Comment by zelph123** [comment]
can't tell you how much i liked cruddiy ..
was wondering if it would do file/image upload ?

**Reply by germain-italic** [reply]
Hi @zelph123 I'm starting to work on this.
